Sources/Sentry/SentryHub.m

```diff
@@ -143,9 +137,9 @@ - (void)closeCachedSessionWithTimestamp:(NSDate *_Nullable)timestamp
                              andLevel:kSentryLogLevelDebug];
             [session endSessionExitedWithTimestamp:timestamp];
         }
+        [self deleteCurrentSession];
+        [client captureSession:session];
```

Comment:
L: We could have a else block here logging, to help us see if things round in a different order during debugging.

Reply:
I don't fully understand your point here sorry. Can you maybe explain it in a different way?